### DISCUSSION TITLE
cmake-init ci-windows and dev-win64 always output to Debug even if CMAKE_BUILD_TYPE is set to Release

### DISCUSSION THREAD

**Discussion by kkuehl** [friendlyanon/cmake-init#107] [Q&A]
cmake-init projects on Windows seem to always output to build/Debug even when CMAKE_BUILD_TYPE is set to Release, instead of the expected build/Release

I also noticed that ci-win64 in CMakePresets.json is the only ci-* that does not set 
```
"cacheVariables" : [
  "CMAKE_BUILD_TYPE": "Release"
]
```
I attempted the above simple modification, and it did not resolve the issue.

**Comment by friendlyanon** [comment]
[Docs](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):

> Specifies the build type on single-configuration generators (e.g. [Makefile Generators](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#makefile-generators) or [Ninja](https://cmake.org/cmake/help/latest/generator/Ninja.html#generator:Ninja)).

On Windows, the generated project is using the Visual Studio 2022 generator, which is a **multi-config generator**. Such generators need the config specified during build.

If you wish to use a single-config generator, Ninja is shipped as part of the Visual Studio build tools (and so is CMake fyi):

```cmd
C:\>call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\vsdevcmd.bat" -arch=amd64 -host_arch=amd64 -no_logo

C:\>where ninja | tail -n1
C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja\ninja.exe

C:\>"C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja\ninja.exe" --version
1.11.0
```
